src/test/resources/maps/50x50_MenuLevel.crf

```diff
@@ -0,0 +1,17 @@
+# Test level with an entity and light
```

Comment:
There are two MenuLevel maps. One in src/test/resources/maps and one in src/main/....I Presume this is because then the lines of the roomreader can be tested with the map on always the same way. Is this correct?